src/doc/tree_externals.dox

```diff
@@ -341,7 +341,7 @@ the first "real" question in the tree.  We can see from the SE questions above i
 applies when the central phone takes values 5 through 19, which are
 various versions of the phone AA; and question is asking whether the pdf-class (key -1)
 has value 0 (i.e. the leftmost HMM-state).  Assuming the answer is "yes", the next question
-is "SE 2 [ 220 221 222 223 ]", which is asking whether the phone to the right is one of various
+is "SE 2 [ 220 221 222 223 ]", which is asking whether the phone to the left is one of various
```

Comment:
In this case, "right" is actually correct.  (2 is the right-most position in the 3-phone window).

Reply:
@danpovey 

I just applied the patch you posted above. I will revert it back tomorrow.

Reply:
I think line 347 is the one which should have been changed.

Reply:
@danpovey 

yes, you're correct!

Fixed.